src/main/java/core/basesyntax/serviceimpl/ReportWriterImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class ReportWriterImpl implements ReportWriter {
+    @Override
+    public void writeReport(String reportName) {
+        String defaultText = "fruit,quantity" + System.lineSeparator();
+        List<String> reportList = Storage.STORAGE
+                .entrySet()
+                .stream()
+                .map(stringIntegerEntry -> stringIntegerEntry.getKey() + ","
+                        + stringIntegerEntry.getValue() + System.lineSeparator())
+                .toList();

```

Comment:
Maybe will be better to make separate reportService that will return List<String> for ReportWriter